sympy/core/evalf.py

```diff
@@ -83,7 +101,13 @@ class PrecisionExhausted(ArithmeticError):
 """
 
 
-def fastlog(x):
+MPF_VALUE_TUPLE = tTuple[int, int, int, int]
+SCALED_ZERO_TUPLE = tTuple[List[int], int, int, int]
+TEMP_RESULT = Any
+# tUnion[tTuple[Optional[MPF_VALUE_TUPLE], Optional[MPF_VALUE_TUPLE], Optional[int], Optional[int]], 'ComplexInfinity']
```

Comment:
There should be comments explaining what these types are and what they mean in practise.

Reply:
They could also have shorter names

Reply:
I move the definitions to the nearest comments that explain them. Shouldn't cause any confusion this time.
https://github.com/eagleoflqj/sympy/blob/afb1af8cd5b2fc876d795f6aaabc22d21c8a81b7/sympy/core/evalf.py#L78-L83